foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -73,7 +74,9 @@ public void setPropertyDao(PropertyDao propertyDao) {
     public Set<Property> loadConfigs(String baseURI, String environmentURI) {
         Set<Property> baseConfigs = loadURI(baseURI);
         Set<Property> environmentConfigs = loadURI(environmentURI);
-
+        boolean isBaseUriValid = baseURI != null && !baseURI.isEmpty();
+        boolean isEnvironmentUriValid = environmentURI != null && !environmentURI.isEmpty();
```

Comment:
C: we probably don't need these since they are already being checked in the loadURI() function and if it's an empty url, the exception should've been thrown.